archivebox/config/__init__.py

```diff
@@ -349,7 +363,10 @@ def load_config_val(key: str,
             raise ValueError(f'Invalid configuration option {key}={val} (expected an integer)')
         return int(val)
 
-    raise Exception('Config values can only be str, bool, or int')
+    elif type is list:
+        return val.split(" ")
```

Comment:
Some arguments have spaces in them that should not be split. It might be better to do `json.loads(val)` and make the user pass lists/objects as json.